### ISSUE TITLE
Uncaught TypeError: Cannot read property 'querySelector' of undefined

### ISSUE THREAD

**Issue by JostCrow** [ericcornelissen/pinned-tabs-for-atom#10]
[Enter steps to reproduce below:]

1. Click the close button.

**Atom Version**: 1.5.0-beta0
**System**: elementary OS
**Thrown From**: [pinned-tabs](https://github.com/ericcornelissen/pinned-tabs-for-atom) package, v0.1.10

### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of undefined

```
At /home/jorik/.atom/packages/pinned-tabs/lib/pinned-tabs.coffee:124

TypeError: Cannot read property 'querySelector' of undefined
    at /home/jorik/.atom/packages/pinned-tabs/lib/pinned-tabs.coffee:124:48
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at PaneContainer.module.exports.PaneContainer.willDestroyPaneItem (/usr/share/atom-beta/resources/app.asar/src/pane-container.js:285:27)
    at Pane.module.exports.Pane.destroyItem (/usr/share/atom-beta/resources/app.asar/src/pane.js:457:17)
    at Pane.module.exports.Pane.destroyActiveItem (/usr/share/atom-beta/resources/app.asar/src/pane.js:444:12)
    at Workspace.module.exports.Workspace.destroyActivePaneItem (/usr/share/atom-beta/resources/app.asar/src/workspace.js:652:35)
    at Workspace.module.exports.Workspace.closeActivePaneItemOrEmptyPaneOrWindow (/usr/share/atom-beta/resources/app.asar/src/workspace.js:691:21)
    at atom-workspace.commandRegistry.add.core:close (/usr/share/atom-beta/resources/app.asar/src/register-default-commands.js:170:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom-beta/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom-beta/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom-beta/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/usr/share/atom-beta/resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/usr/share/atom-beta/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
  9x -0:59.5.0 core:backspace (atom-text-editor.editor.is-focused)
 37x -0:56.7.0 core:move-right (atom-text-editor.editor.is-focused)
     -0:50.8.0 core:save (atom-text-editor.editor.is-focused)
     -0:47.6.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:46.4.0 core:select-right (atom-text-editor.editor.is-focused)
     -0:45.7.0 core:cut (atom-text-editor.editor.is-focused)
 11x -0:44.8.0 core:undo (atom-text-editor.editor.is-focused)
     -0:39.9.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused.hyperclick)
     -0:38.6.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:38.6.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:37.9.0 core:redo (atom-text-editor.editor.is-focused.hyperclick)
  2x -0:32.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:31.5.0 core:paste (atom-text-editor.editor.is-focused.hyperclick)
     -0:30.8.0 core:save (atom-text-editor.editor.is-focused)
     -0:03.6.0 core:copy (atom-text-editor.editor.is-focused)
     -0:02.5.0 core:close (atom-text-editor.editor.is-focused.hyperclick)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "sass-autocompile",
      "language-python"
    ],
    "openEmptyEditorOnStart": false,
    "projectHome": "/srv/sites/",
    "themes": [
      "graphite-ui",
      "material-design-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
MagicPython, v0.5.7
apm-publish, v0.2.8
atom-beautify, v0.28.21
atom-bootstrap4, v1.4.0
atom-jade, v0.3.0
atom-ungit, v0.5.3
autocomplete-python, v1.4.4
autoupdate-packages, v1.3.1
color-picker, v2.1.0
coverage, v0.6.1
custom-title, v1.0.1
django-templates, v0.5.0
fonts, v1.0.0
graphite-ui, v0.15.1
highlight-line, v0.11.0
hyperclick, v0.0.35
icon-fonts, v1.10.1
linter, v1.11.3
linter-pep8, v1.2.1
linter-pylint, v1.2.0
linter-sass-lint, v0.6.3
material-design-syntax, v1.7.0
minimap, v4.19.0
minimap-find-and-replace, v4.4.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-linter, v1.1.1
minimap-pigments, v0.1.7
nuclide-hg-repository, v0.0.35
pigments, v0.20.0
pinned-tabs, v0.1.10
python-tools, v0.6.7
selection-highlight, v0.1.4
set-syntax, v0.3.0
seti-icons, v0.4.5
soft-wrap-indicator, v0.7.0
term3, v0.21.0
terminal-plus, v0.14.5
tester, v0.0.2
tidy-tabs, v0.3.0

# Dev
No dev packages
```

**Comment by ericcornelissen** [comment]
Could you specify what close button I have to click to reproduce this bug ?

**Comment by ericcornelissen** [comment]
I added a initial fix for this bug in 9a42ad9 and published that to APM. Let me know if it happens again.

**Comment by JostCrow** [comment]
Ok Thanks.
